src/UI/Settings/Notifications/Edit/NotificationEditView.js

```diff
@@ -11,14 +11,17 @@ var view = Marionette.ItemView.extend({
     template : 'Settings/Notifications/Edit/NotificationEditViewTemplate',
 
     ui : {
-        onDownloadToggle : '.x-on-download',
-        onUpgradeSection : '.x-on-upgrade',
-        tags             : '.x-tags'
+        onDownloadToggle             : '.x-on-download',
+        onUpgradeSection             : '.x-on-upgrade',
+        tags                         : '.x-tags',
+        indicator                    : '.x-indicator',
+        authorizedNotificationButton : '.AuthorizeNotification'
```

Comment:
I was a little unsure how to handle that. Right now its a FieldAttribute of type button, so the class name is dependant on the field name. Should I just change it to be a boolean type field or something? Hard code the class?